README.md

```diff
@@ -180,6 +180,22 @@ is parallel. It is based on the work of:
 
 [Lagaris, Isaac E., Aristidis Likas, and Dimitrios I. Fotiadis. "Artificial neural networks for solving ordinary and partial differential equations." IEEE Transactions on Neural Networks 9, no. 5 (1998): 987-1000.](https://arxiv.org/pdf/physics/9705023.pdf)
 
+## Solving Kolmogorov Equations with Neural Networks
```

Comment:
needs more information. What's the form of the equations? The relation to the SDE?